src/kepler_model/util/train_types.py

```diff
@@ -45,8 +45,15 @@
     "KNeighborsRegressorTrainer",
     "LinearRegressionTrainer",
     "SVRRegressorTrainer",
-    "XgboostFitTrainer",
 ]
+have_xgboost = True
+try:
+    import xgboost
+except:
+    have_xgboost = False
+if have_xgboost:
+    no_weight_trainers.append("XgboostFitTrainer")
```

Comment:
```suggestion
try:
    import xgboost
    no_weight_trainers.append("XgboostFitTrainer")
except ImportError:
    pass
```

Reply:
On second thought, we don't want import xgboost rather only need to know if xgboost can be imported, hence could you please use `importlib.unit.find_spec` ?